### DISCUSSION TITLE
Update to Schema.org site

### DISCUSSION THREAD

**Discussion by rvguha** [schemaorg/schemaorg#3543] [General]
The schema.org site has not had a UI refresh since we first launched in 2011.

Adriana Olmos and Carolyn Au have worked on a refresh. An early version can be found at https://1-dot-schemaorgtest-425516.uc.r.appspot.com/

Please send feedback.

**Comment by dwoydig** [comment]
Hi there,

just a minor suggestion that would make my life easier:
on https://validator.schema.org , it will detect the schema type (for example "BreadcrumbList"). Could we have a link to the docs page for the item, so it's easy to find examples and possible options for this schema?
